data.json

```diff
@@ -316,10 +316,10 @@
   },
   {
     "name": "Operation Code",
-    "difficulty": "easy",
-    "description": "Close 3 issues, get a t-shirt. Make 2 PRs and get stickers.",
+    "difficulty": "medium",
+    "description": "Resolve 3 issues, get a t-shirt. Merge 2 PRs and get stickers.",
     "reference": "https://github.com/OperationCode/START_HERE",
```

Comment:
I guess we can change that image but we would need to rehost it on imgur for example.

Reply:
Sorry I merged early. 🙈